brieflow_workflows_env.yml

```diff
@@ -13,3 +13,4 @@ dependencies:
   - numpy
   - pip:
       - -e .
+      - snakemake-executor-plugin-slurm
```

Comment:
It looks like there is an actively maintained [conda package](https://anaconda.org/bioconda/snakemake-executor-plugin-slurm) for this. Are we able to move to that package for consistent package management? I've also been reading in the snakemake docs that conda is the preferable package manager where possible.

Definitely worth making sure that the environment works if we do end up going with the conda verison though.